pkg/v22/key/types.go

```diff
@@ -5,8 +5,13 @@ type AppSpec struct {
 	App     string
 	Catalog string
 	Chart   string
-	// Whether app is installed for clusterapi clusters only
-	ClusterAPI      bool
+	// Whether app is installed for clusterapi clusters only.
+	ClusterAPIOnly bool
+	// ConfigMapName overrides the name, otherwise the cluster values configmap
+	// is used.
+	ConfigMapName string
```

Comment:
This is prep for migrating Ingress Controller in a later PR. We need to use a different configmap with the node count until we can enable HPA.